FEM/rf_st_new.cpp

```diff
@@ -3371,16 +3379,25 @@ void CSourceTerm::SetNOD()
  11/2007 JOD
  last modification:
  **************************************************************************/
-void CSourceTermGroup::SetSurfaceNodeVector(Surface* m_sfc, std::vector<long>& sfc_nod_vector)
+void CSourceTermGroup::SetSurfaceNodeVector(Surface* m_sfc, std::vector<long>& sfc_nod_vector, CSourceTerm* st)
```

Comment:
You can use a double argument here instead of `CSourceTerm*` For example,
 ```
 ::SetSurfaceNodeVector(Surface* m_sfc, std::vector<long>& sfc_nod_vector, const double tolerance)
```